docs/source/models/supported_models.md

```diff
@@ -56,9 +56,13 @@ If it is `TransformersModel` then it means it's based on `transformers`!
 
 #### Supported features
 
-##### LORA and quantization
+##### Quantization
 
-Both are not supported yet! Make sure to open an issue and we'll work on this together with the `transformers` team!
+Transformers fallback has supported most of available quantization in vllm (except GGUF). See [Quantization page](#quantization-index) for more information about supported quantization in vllm.
+
+##### LoRA
+
+LoRA hasn't supported yet! Make sure to open an issue and we'll work on this together with the `transformers` team!
```

Comment:
We have had issues in the past where people have assumed this wasn't specific to the Transformers fallback. I think we should make that clearer